wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -1055,10 +1044,8 @@ private void handleBlockchainStateNotification(BlockchainState blockchainState)
                 nm.cancel(Constants.NOTIFICATION_ID_BLOCKCHAIN_SYNC);
             } else if (blockchainState.getReplaying() || syncing) {
                 //Shows ongoing notification when synchronizing the blockchain
-                Notification notification = createNetworkSyncNotification(blockchainState);
-                if (notification != null) {
-                    nm.notify(Constants.NOTIFICATION_ID_BLOCKCHAIN_SYNC, notification);
-                }
+                Notification notification = createNetworkSyncNotification();
```

Comment:
We still need to pass the blockchainState here so the notification will be updated with the sync state.